rllib/tests/test_rllib_train_and_evaluate.py

```diff
@@ -96,12 +96,23 @@ def learn_test_plus_evaluate(algo: str, env="CartPole-v1"):
         print("Saving results to {}".format(tmp_dir))
 
         rllib_dir = str(Path(__file__).parent.parent.absolute())
+
+        # This is only supported without RLModule API. See AlgorithmConfig for
+        # more info. We need to prefetch the default config that will be used when we
+        # call rllib train here to see if the RLModule API is enabled.
+        algo_cls = get_trainable_cls(algo)
+        config = algo_cls.get_default_config()
+        if config._enable_rl_module_api:
+            eval_ = ', \\"evaluation_config\\": {}'
```

Comment:
We _need_ to but an empty evaluation config here because otherwise we run into an error when calling rllib with popen further down. This is because inside the `rllib evaluate` script we attempt to access a command line argument `evaluation_config `.